lib/posix-process/signal/signal.h

```diff
@@ -223,4 +223,38 @@ int pprocess_signal_thread_do(int tid, int signum, siginfo_t *siginfo);
 int pprocess_signal_send(struct posix_process *proc, int signum,
 			 siginfo_t *siginfo);
 
+void uk_signal_deliver(struct uk_syscall_ctx *usc);
+
+/* Jump to signal handler
+ *
+ * Trampoline to jump from unikraft to userspace context.
+ *
+ * @param signum signal number
+ * @param si     data to pass as the 2nd parameter to the handler
+ *               if SA_SIGINFO is set, or NULL
+ * @param ctx    user context to pass as the 3rd parameter to
+ *               the handler if SA_SIGINFO is set, or NULL
+ * @param sp     the user stack pointer address to switch to
+ *               before calling the handler
+ */
+void pprocess_signal_arch_jmp_handler(int signum, siginfo_t *si,
+				      ucontext_t *ctx,
+				      void *handler, void *sp);
+
+/* Populate POSIX u_context from uk syscall context */
```

Comment:
Any reason the functions below don't have the `@` comments like the one above too?

Reply:
Same reason as in the previous comment, in general I didn't use doxygen for private functions, but for this particluar functions args seemed to deserve some longer explanation.